superset/config.py

```diff
@@ -748,6 +748,10 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 # note: index option should not be overridden
 CSV_EXPORT = {"encoding": "utf-8"}
 
+#It Is used for enable/disable the index sequence for csv which came at email-time,
+#rearly someone want the index in csv so by-default it is false
+CSV_INDEX={"index":False}
+
```

Comment:
@betodealmeida Your all changes looks good and thanks for responce. 
But According to this comment :
https://github.com/apache/superset/issues/22981#issuecomment-1440410419
I would thing we don't have to give config `opt true/false` , instead we can make this  `processed_df.to_csv(buf, index=config["CSV_INDEX"]) ` to processed_df.to_csv(buf, index=False) Because it is really a bug.
what do you think ? I like to listen your thougts regarding to this.
@Antonio-RiveroMartnez

Reply:
Hey @rohitpawar2811 ! we could use `True/False` in case some users got used to the `row_number` already? as @betodealmeida said, we can change that in future releases along addressing the other problems listed in the issue. Let's add some tests for both cases tho. Thanks again!